### ISSUE TITLE
Validation.ErrorTemplate usage seems broken when ModernWpfUI styles are enabled

### ISSUE THREAD

**Issue by mgnslndh** [Kinnara/ModernWpf#232]
I know there is no built in template for `Validation.ErrorTemplate`  but if I provide my own I would assume it would work. However, it seems like something breaks when using a  `Validation.ErrorTemplate`  and ModernWpfUI.

I have a [sample repository](https://github.com/mgnslndh/ModernWpfUIValidation) showcasing the problem.

# Validation with ModernWpfUI

![WithStyle](https://user-images.githubusercontent.com/1139118/101895375-beb45a80-3ba7-11eb-81ab-dd25ef6095ce.png)

# Validation without ModernWpfUI

![WithoutStyle](https://user-images.githubusercontent.com/1139118/101895382-c116b480-3ba7-11eb-886a-171bfd1e2f5d.png)

**Comment by Kinnara** [comment]
Thanks for reporting. Seems to be the same issue as [this one](https://github.com/Kinnara/ModernWpf/issues/188)? I'll try to think of a solution in the near feature. In the meantime, you can use the workaround given in that issue.

**Comment by mgnslndh** [comment]
Yes, these two seems very similar. I will try the workaround and report back if it is working.

**Comment by Kinnara** [comment]
Closing this as a duplicate of #188, as identified in the existing discussion and confirmed by the reporter. I’m keeping #188 open as the canonical issue and will reproduce the exact custom `Validation.ErrorTemplate` binding case there against the current 1.x templates.
